frontend/src/app/index.js

```diff
@@ -1,5 +1,5 @@
 import 'babel-polyfill/browser';
-import 'l20n/dist/compat/web/l20n';
```

Comment:
In previous versions of l20n there were two directories, `dist/bundle/web` and `dist/compat/web` containing the original and the transpiled builds respectively. l20n is now transpiled using Babel's Latest preset by default and the result is `dist/web/l20n.js` which is also specified as the `main` field in `package.json`.